apps/condo/domains/notification/templates.js

```diff
@@ -1,278 +1,176 @@
-                    text: `
-                    Добрый день! \n
-                    Чтобы задать новый пароль к платформе Doma.ai, вам просто нужно перейти по сслыке.\n
-                    ${serverUrl}/auth/change-password?token=${token}
-                `,
-                }
-            }
-        } else if (transport === SMS_TRANSPORT) {
-            if (message.lang === 'en') {
-                return {
-                    text: `Click to the link to set new password: ${serverUrl}/auth/change-password?token=${token}`,
-                }
-            } else if (message.lang === 'ru') {
-                return {
-                    text: `
-                        Перейдите по сслыке для изменения пароля: ${serverUrl}/auth/change-password?token=${token}
-                    `,
-                }
-            }
+        if (templatePathText) {
+            ret.text = nunjucks.render(templatePathText, { message, env })
         }
-    }
 
-    if (message.type === SMS_VERIFY_CODE_MESSAGE_TYPE) {
-        const { smsCode } = message.meta
-        if (message.lang === 'en') {
-            return {
-                subject: 'Verify code',
-                text: `Code: ${smsCode}`,
-            }
-        } else if (message.lang === 'ru') {
-            return {
-                subject: 'Код',
-                text: `Код: ${smsCode}`,
-            }
+        if (templatePathHtml) {
+            ret.html = nunjucks.render(templatePathHtml, { message, env })
         }
-    }
 
-    if (message.type === DEVELOPER_IMPORTANT_NOTE_TYPE) {
-        const { data, type } = message.meta
+        return ret
+    },
+    [TELEGRAM_TRANSPORT]: function ({ message, env }) {
+        throw new Error('No Telegram transport yet')
+    },
+    [PUSH_TRANSPORT]: function ({ message, env }) {
+        const { lang, meta } = message
         return {
-            subject: String(type),
-            text: JSON.stringify(data),
+            notification: {
+                title: i18n(translationStringKeyForPushTitle(message.type), { lang, meta }),
+                body: nunjucks.render(getTemplate(message.lang, message.type, PUSH_TRANSPORT), { message, env }),
+            },
+            data: get(message, ['meta', 'pushData'], null),
+            userId: get(message, ['meta', 'userId'], null),
         }
-    }
+    },
+}
 
-    if (message.type === CUSTOMER_IMPORTANT_NOTE_TYPE) {
-        const { data } = message.meta
+async function renderTemplate (transport, message) {
+    if (!MESSAGE_TRANSPORTS.includes(transport)) {
+        throw new Error('unexpected transport argument')
+    }
 
-        return {
-            subject: 'Новая организация. (СББОЛ)',
-            text: `
-                Название: ${get(data, ['organization', 'name'])},
-                ИНН: ${get(data, ['organization', 'meta', 'inn'])},
-            `,
-        }
+    if (!Object.keys(MESSAGE_TRANSPORTS_RENDERERS).includes(transport)) {
```

Comment:
Please move `Object.keys(MESSAGE_TRANSPORTS_RENDERERS)` to static constant cause it contains cycles under the hood.

Reply:
Done.